app/src/main/java/org/wikipedia/descriptions/DescriptionEditFragment.kt

```diff
@@ -545,6 +545,11 @@ class DescriptionEditFragment : Fragment() {
         const val SUGGESTED_EDITS_TRANSLATE_DESC_COMMENT = "#suggestededit-translate-desc $SUGGESTED_EDITS_UI_VERSION"
         const val SUGGESTED_EDITS_ADD_CAPTION_COMMENT = "#suggestededit-add-caption $SUGGESTED_EDITS_UI_VERSION"
         const val SUGGESTED_EDITS_TRANSLATE_CAPTION_COMMENT = "#suggestededit-translate-caption $SUGGESTED_EDITS_UI_VERSION"
+        const val SUGGESTED_EDITS_IMAGE_TAGS_COMMENT = "##suggestededit-add-tag $SUGGESTED_EDITS_UI_VERSION"
+        const val WATCHLIST_UNDO_COMMENT = "#watchlist-undo"
+        const val WATCHLIST_ROLLBACK_COMMENT = "#watchlist-rollback"
+        const val DIFF_UNDO_COMMENT = "#diff-undo"
+        const val DIFF_ROLLBACK_COMMENT = "#diff-rollback"
```

Comment:
Do all of the above new comments belong in `DescriptionEditFragment.kt`, if they don't relate to description editing? Can we put them in more relevant places?

Reply:
Made this change... but was wondering if we should do the opposite and have all the comment tags at one place... makes answering analytics questions easier... even if it means a new section in `Constants` or a whole new file of its own..